wpeverest/user-registration
# UR-1218 Fix -Avatar/profile pictures files are left behind even if the user is deleted

### All Submissions:






### Changes proposed in this Pull Request:



Previously Avatar/profile pictures files are left behind in /wp-content/uploads/user_registration_uploads/profile-pictures/. even if the user is deleted (with higher resolution pictures). This PR resolve that bug

### How to test the changes in this Pull Request:
Note: use higher resolution pictures or disable crop for profile pictures

Case I:
1.Register user with profile picture.
2.Remove user from user registration pro->users->delete.\
Case II:
1.Register user with profile picture again.
2.Remove user from users->delete (default users of wp).

After Case I or Case II:
 check  folder /wp-content/uploads/user_registration_uploads/profile-pictures/ recently deleted users all image should be removed and from wp_postmeta _wp_attached_file and _wp_attachment_metadata  meta key should be removed of particular users deleted








### Types of changes:




### Other information:




### Changelog entry

>  Fix -Avatar/profile pictures files are left behind  even if the user is deleted
